### DISCUSSION TITLE
Log file for each individual process

### DISCUSSION THREAD

**Discussion by VishalGujela** [gabime/spdlog#2952] [Q&A] [answered]
Hi,
I was just curious to know that can we use spdlog to write in multiple files in a multithread application.
So that i will be able to see separate log files each individual process im running.
Thanks

**Comment by tt4g** [comment, marked as answer]
You can create a logger with multiple sinks.
See: https://github.com/gabime/spdlog/wiki/2.-Creating-loggers#creating-loggers-with-multiple-sinks

Since spdlog does not support writing to a single file from multiple processes, you should avoid file name conflicts for each process.
